ceph_iscsi_config/client.py

```diff
@@ -274,66 +289,53 @@ def try_disable_auth(self, tpg):
 
         tpg.set_attribute('authentication', '0')
 
-    def configure_auth(self, chap_credentials, chap_mutual_credentials, target_config):
+    def configure_auth(self, username, password, mutual_username, mutual_password, target_config):
         """
         Attempt to configure authentication for the client
         :return:
         """
 
         auth_enabled = False
-        if '/' in chap_credentials:
-            client_username, client_password = chap_credentials.split('/', 1)
+        if username and password:
             auth_enabled = True
-        elif not chap_credentials:
-            client_username = ''
-            client_password = ''
-            chap_credentials = "/"
 
-        if '/' in chap_mutual_credentials:
-            client_mutual_username, client_mutual_password = chap_mutual_credentials.split('/', 1)
+        if mutual_username and mutual_password:
```

Comment:
Nit: if ```username``` / ```password``` are empty, it would be an error to attempt to configure mutual auth, correct? Perhaps a check that could be added to ```valid_credentials```?